spaceinvaders.py

```diff
@@ -289,7 +289,7 @@ def __init__(self):
 		self.gameOver = False
 		self.enemyposition = 65
 
-	def reset(self, score, lives):
+	def reset(self, score, lives, newGame=False):
```

Comment:
Variables should be camel_case

Reply:
Whoops - I'm used to my work PEP standards 🤣 This is fine since there's camelCase variables in this file.